help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -77,10 +75,11 @@ To configure details for the destination, fill in the required and optional fiel
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
 *  **[!UICONTROL Amazon Ads Advertiser ID]**: Select the ID for the target Amazon Ads account used for the destination.
+*  **[!UICONTROL Advertiser Region]**: Select the appropriate region in which your AdvertiserId is hosted. For more information on the marketplaces supported by each region, visit our [documentation](https://advertising.amazon.com/API/docs/en-us/info/api-overview#api-endpoints).
```

Comment:
Line 78: Shouldn't it be "Select the appropriate region in which your Advertiser account is hosted." ? Sounds strange to host an ID.

If you want to keep referring to the Advertiser ID, please use "Advertiser ID" instead of "AdvertiserId".

Reply:
Done - changed "AdvertiserId" to "Advertiser".